cli/azd/internal/repository/app_init.go

```diff
@@ -1107,12 +1112,61 @@ func promptSpringBootVersion(console input.Console, ctx context.Context) (string
 	}
 }
 
+func promptMissingEventHubsNameOrExit(console input.Console, ctx context.Context, eventHubs *appdetect.AzureDepEventHubs) {
+	switch eventHubs.FromDependency {
+	case appdetect.SpringIntegrationEventHubs, appdetect.SpringMessagingEventHubs, appdetect.SpringKafka:
+		eventHubsName, err := promptEventHubsName(console, ctx)
+		if err != nil {
+			console.Message(ctx, fmt.Sprintf("Error happened when prompt eventhubs name: %s.", err))
+			os.Exit(-1)
+		}
+		eventHubs.EventHubsNamePropertyMap = map[string]string{
+			string(eventHubs.FromDependency): eventHubsName,
+		}
+	case appdetect.SpringCloudStreamEventHubs, appdetect.SpringCloudStreamKafka:
+		promptMissingPropertyAndExit(console, ctx, "spring.cloud.stream.bindings.<binding name>.destination")
+		os.Exit(0)
+	case appdetect.SpringCloudEventHubsStarter:
+		promptMissingPropertyAndExit(console, ctx, "spring.cloud.azure.eventhubs.event-hub-name")
+		os.Exit(0)
+	}
+}
+
 func promptMissingPropertyAndExit(console input.Console, ctx context.Context, key string) {
 	console.Message(ctx, fmt.Sprintf("No value was provided for %s. Please update the configuration file "+
 		"(like application.properties or application.yaml) with a valid value.", key))
 	os.Exit(0)
 }
 
+func promptEventHubsName(console input.Console, ctx context.Context) (string, error) {
+	for {
+		eventHubsName, err := console.Prompt(ctx, input.ConsoleOptions{
+			Message: "Input the name of Azure EventHubs (Not EventHubs namespace name):",
+			Help:    "Hint: Azure EventHubs Name, not EventHubs namespace name",
+		})
+		if err != nil {
+			return "", err
+		}
+		if IsValidEventhubsName(eventHubsName) {
+			return eventHubsName, nil
+		} else {
+			console.Message(ctx, "Invalid eventhubs name. Please choose another name.")
```

Comment:
Add more information about what is a valid name.

Reply:
Updated.